src/Kdump.ycp

```diff
@@ -280,6 +280,15 @@ global void SetModified() {
     y2debug("modified=%1",modified);
 }
 
+/**
+ * Determine whether boot section is Xen or not
+ * It relies on section_pos
+ *
+ * @return boolean
+ */
+boolean is_xen () {
+	return (section_pos != -1);
```

Comment:
NP: comparing to -1 is OK, but in such situations I rather tend to write `> 0` to handle also other (though unlikely) negative values. It's more robust and reliable...